README.md

```diff
@@ -156,52 +165,35 @@ with the following content:
 }
 ```
 
-#### * Important - Canonical URLs
-The `url` element in a conformance resource is called the canonical URL and used as a
-unique identifier for profiles on the server. The server does not check to
-see if the URL actually resolves to anything, but it does check to see if
-the `url` attribute is present and the value is unique among others loaded
-on the server.
-
 ### Example Resources
-Next create an example resource in the example resources directory
+Next create an example resource in the example resources directory:
 
-`./site_root/source/examples/research-study-example.json`
+`./site_root/input/resources/examples/rs-001.json`
```

Comment:
For the abbreviation of an example resource, do we want to abbreviate the base profile or custom profile? I mean, `'rs'` vs `'sd'`?

Reply:
I don't think this matters that much since its just an example resource in the model. But I guess if we pick one it makes more sense to do `rs-001` otherwise `sd-001` could refer to either a `participant` instance or a `study` instance.

Reply:
oh, I was indicating SD as in Study (like we abbreviate PT, BS, etc.).

Reply:
Oh I see. Hmmm, yea sure let's us our Kids First abbreviations. I'll update the README 👍